wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -61,8 +62,9 @@ Mapsets of songs that originated from other rhythm games often borrow that game'
 - ![](/wiki/shared/diff/insane-s.png) Expert
 - ![](/wiki/shared/diff/expert-s.png) Challenge
 
-### DJMax
+### DJMAX
 
+- ![](/wiki/shared/diff/easy-s.png) EZ
```

Comment:
should write out the full names as well

Reply:
![image](https://user-images.githubusercontent.com/16955881/67843829-19bff500-fb40-11e9-9920-9dd9ba356c2f.png)
they all use short ones